README.md

```diff
@@ -112,7 +112,7 @@ talis jobs get --id job-20240315-123456
 Example configuration (create.json):
 ```json
 {
-    "name": "talis",
+    "instance_name": "talis",
```

Comment:
it would be more clear to call this `instance_prefix_name` or just `prefix_name`

Reply:
imo calling this a `prefix_name` is more confusing. 
Prefix and Suffix call attention to that part of the name that is secondary. 
In this naming scheme, we are adding incremental suffixes to a name.
We are not prefixing some other name.